src/functional.ts

```diff
@@ -75,27 +57,12 @@ export function functionalInterfaceFactory({
    */
   function verifyAggregate(publicKeys: PublicKeyArg[], message: Uint8Array, signature: SignatureArg): boolean {
     try {
-      validateBytes(message, "message");
-
-      const pks: PublicKey[] = [];
```

Comment:
is it not necessary to `validateByutes` for public keys anymore?

Reply:
I found it's implemented for `PublicKey.convertToPublicKeyType` of `herumi` but not for `blst-native`